src/Microsoft.ML.Data/DataLoadSave/Text/TextLoaderStatic.cs

```diff
@@ -2,15 +2,16 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.Collections.Generic;
 using Microsoft.ML.Core.Data;
-using Microsoft.ML.StaticPipe;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
 using Microsoft.ML.StaticPipe.Runtime;
+using System;
+using System.Collections.Generic;
 
-namespace Microsoft.ML.Runtime.Data
+namespace Microsoft.ML.StaticPipe
 {
-    public sealed partial class TextLoader
+    public static class TextLoaderStatic
```

Comment:
>TextLoaderStatic [](start = 24, length = 16)

I expect this will go into the M.ML.StaticPipe assembly, right?

Reply:
yeap. that's the intent  (once i verify build are passing)

---
In reply to: [243091763](https://github.com/dotnet/machinelearning/pull/1930#discussion_r243091763) [](ancestors = 243091763)